api/src/main/java/edu/cornell/mannlib/vitro/webapp/application/VitroHomeDirectory.java

```diff
@@ -210,4 +217,79 @@ public String toString() {
 		}
 	}
 
+    /**
+     * Populates home directory with home files, excluding the rdf directory
+     */
+    public void populate() {
+        File vhdDir = getPath().toFile();
+
+        if (!vhdDir.isDirectory() || vhdDir.list() == null) {
+            throw new RuntimeException("Application home dir is not a directory! " + vhdDir);
+        }
+        
+        if (!vhdDir.canWrite()) {
+            throw new RuntimeException("Application home dir is not writable! " + vhdDir);
+        }
+        try {
+            copy(vhdDir);
+        } catch(Exception e) {
+            log.error(e, e);
+            throw new RuntimeException("Failed to copy home files! " + vhdDir);
```

Comment:
The throwable could be added to the RuntimeException instantiation and the error log can be removed.

Reply:
Done.